pyproject.toml

```diff
@@ -15,3 +15,69 @@ exclude = '''
   | [a-z_]+/migrations
 )/
 '''
+
```

Comment:
I moved every configuration file here. Only pre-commit and prospector don't support pyproject.toml yet